.gitignore

```diff
@@ -11,6 +11,8 @@
 .pub-cache/
 .pub/
 build/
+.pubspec.lock
```

Comment:
I think I saw a while back that this shouldn't be added if we want to follow dart guideline. Is there a reason for the CI it's there? Found the [stack overflow thread I'm referring to](https://stackoverflow.com/questions/16136739/in-dart-and-pub-should-i-add-pubspec-lock-to-my-gitignore) that explain this. Don't know if it's the same for package-lock.json though.